apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/APIServlet.java

```diff
@@ -222,6 +224,12 @@ private void process(HttpServletRequest req, HttpServletResponse resp) throws IO
             }
         } catch (ParameterException e) {
             response = e.getErrorResponse();
+        } catch (BlockNotFoundException e){
```

Comment:
You can collapse this block and move exception handling  to `AplException | RuntimeException e` block

Reply:
I've split those exceptions in purpose. There isn't the stacktrace output in the BlockNotFound catch block.